test/test_script.jl

```diff
@@ -130,7 +130,7 @@ for key in  [:InvertedCoax, :BEGe, :Coax, :CGD, :Spherical]
     savefig(joinpath(outputdir, "$(key)_4_charge_drift"))
 
     # signals[:, 2] *= -1
-    plot(event.waveforms, size = (1200, 600), lw = 1.5)
+    plot([event.waveforms...], size = (1200, 600), lw = 1.5)
```

Comment:
Is there any way to plot the waveforms without getting an error?
I think the problem is that `event.waveforms` is of type `Union{Missing, Vector{RDWaveform}}` such that it is not clear to the `plot` method whether there are waveforms stored already?

Reply:
We should try not to have a possible `missing` in there.